logger.go

```diff
@@ -34,15 +33,10 @@ type Shuttle struct {
 
 // NewShuttle creates a shuttle client from a Config.
 func NewShuttle(cfg *LoggerConfig) (*Shuttle, error) {
-	token, ok := cfg.URL.User.Password()
-	if !ok {
-		return nil, errors.New("no logplex password provided")
-	}
-
 	// Create shuttle config.
 	conf := shuttle.NewConfig()
 	conf.LogsURL = cfg.URL.String()
-	conf.Appname = token
+	conf.Appname = "heroku"
```

Comment:
We do log as `app`

`2016-03-25T20:03:18Z app[postgres.4905]: [GREGBUREK] connection authorized: user=postgres database=da0a8rcolsmai1 SSL enabled (protocol=TLSv1.2, cipher=ECDHE-RSA-AES256-GCM-SHA384, compression=off)`

This seems like a small change that hides other problems. +0 from me, and I would like another pair of eyes from @heroku/department-of-data